docs/advanced/api.rst

```diff
@@ -14,12 +14,24 @@ Installation
 Fedora/CentOS
 -------------
 
-To allow remote access to the control agent REST API, and for agent connections,
+To enable Flocker control service.
+
+.. task:: enable_flocker_control
+   :prompt: [root@control-node]#
+
+The control service needs to accessible remotely.
+To configure firewalld to allow access to the control service REST API, and for agent connections,
 
 .. task:: open_control_firewall
 
+(On AWS, an external firewall is used instead, which will need to be configured similarity).
```

Comment:
Is there a follow up issue to document this configuration?

If not, please make one.
If so, please either link to that here (in a comment) or mention this section on that issue as something to change.